.github/workflows/main.yml

```
@@ -13,28 +13,26 @@ jobs:
           latest: true
           fileName: "*"
       - name: Unzip
-        uses: montudor/action-zip@v1
-        with:
-          args: unzip -qq *.zip -d executables
+        run: unzip '*.zip' -d executables
       - name: Run Interoperability script
         run: |
           python3 -m venv .venv
           source .venv/bin/activate
           pip install -r requirements.txt
           cd executables
-          for i in ./* ; do for j in ./*; do python3 ./../interoperability_report.py -P $i -S $j -o=./../general_report.xml; done; done
+          for i in ./* ; \
+          do for j in ./*; \
+          do python3 ./../interoperability_report.py -P $i -S $j -o=./../junit_interoperability_report.xml; \
+          done; \
+          done
```

```suggestion
          for i in ./* ; \
            do for j in ./*; \
              do python3 ./../interoperability_report.py -P $i -S $j -o=./../junit_interoperability_report.xml; \
            done; \
          done
```

Not sure this can be done, but it this works, then we should add some spaces for the internal for loops

---

okey